fluentbit.tf

```diff
@@ -58,3 +68,34 @@ module "fluentbit_container_definition" {
     var.firelens.container_definition
   ]
 }
+
+data "aws_iam_policy_document" "fluent_bit_config_access" {
+  count = var.firelens.enabled && var.task_role_arn == "" ? 1 : 0
+
+  statement {
+    actions = [
+      "s3:GetObject",
+      "s3:GetBucketLocation"
+    ]
+
+    resources = [
+      "arn:aws:s3:::config-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}/ecs/fluent-bit/*",
+      "arn:aws:s3:::config-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}",
+    ]
+  }
+}
```

Comment:
if we include the default config (see above), we could:

- restrict `s3:GetBucketLocation` to `arn:aws:s3:::config-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}`
- restrict `s3:GetObject` to the concrete files instead of `*`

?